language/ir-testsuite/tests/signer/borrow_address.mvir

```diff
@@ -0,0 +1,11 @@
+//! account: alice, 90000
+//! new-transaction
+//! sender: alice
+
+import 0x0.Signer;
+main(s: &signer) {
+    let x: address;
+    x = {{alice}};
+    assert(Signer.borrow_address(move(s)) == &x, 42);
+    return;
+}
```

Comment:
Do we need both Move and mvir tests for these? I have found it painful to maintain two copies of the same test when we do this elsewhere...

Reply:
I could move them just to Move, sure